moveit_msgs/RobotTrajectory old_plan
---
moveit_msgs/RobotTrajectory new_plan
